presto-docs/src/main/sphinx/release/release-0.243.rst

```diff
@@ -0,0 +1,48 @@
+=============
+Release 0.243
+=============
+
+**Highlights**
+==============
+* Add function ``truncate(x, n)``.
+* Add aggregation function ``approxMostFrequent``.
+* Add session property ``check_access_control_on_utilized_columns_only``, which, when enabled, only performs access control checks on columns that would actually be required to produce the query output, ignoring columns that are referenced in the query, but are not required to compute the query results.
+* Add session property ``remote_functions_enabled`` to configure whether remote functions are allowed.
+
+**Details**
+===========
+
+General Changes
+_______________
+* Add function ``truncate(x, n)``.
+* Add aggregation function ``approxMostFrequent``.
+* Fix query failure when explaining ``Use`` statement.
+* Add session property ``check_access_control_on_utilized_columns_only``, which, when enabled, only performs access control checks on columns that would actually be required to produce the query output, ignoring columns that are referenced in the query, but are not required to compute the query results.
+* Add session property ``remote_functions_enabled`` to configure whether remote functions are allowed.
+* Remove config property ``optimizer.optimize-full-outer-join-with-coalesce`` and the corresponding session property ``optimize_full_outer_join_with_coalesce``. The feature will always be enabled.
+* Remove deprecated configuration properties ``grouped-execution-for-aggregation-enabled`` and ``grouped-execution-for-join-enabled`` and their corresponding session properties ``grouped_execution_for_aggregation`` and ``grouped_execution_for_join``.  These have been replaced by a single configuration property ``grouped-execution-enabled`` and session property ``grouped-execution``.
+* Remove deprecated configuration property ``dynamic-schedule-for-grouped-execution`` and session property ``dynamic_schedule_for_grouped_execution``.  No replacement is needed.
+
+Verifier Changes
+________________
+* Improve verifier handling of broadcast memory limit by removing it, if it is set via session property to avoid CBO broadcast induced failures on smaller verifier clusters.
+* Add support to verify ``CREATE VIEW`` and ``CREATE TABLE`` queries.
+
+Geospatial Changes
+__________________
+* Add support to allow geometries outside of the spatial partitioning to match in a partitioned spatial join.
+
+Hive Changes
+____________
+* Add caching module for Parquet metadata.
```

Comment:
what's this mean from a user perspective?

Reply:
@ClarenceThreepwood @shixuan-fan

Reply:
How about rephrasing this to:

`Add support for Parquet file metadata caching`

The related config is `parquet.metadata-cache-enabled`, `parquet.metadata-cache-size` and `parquet.metadata-cache-ttl-since-last-access`, but I'm not sure if we want to go to this level of detail after reading other bullets.